portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+public final class Comment{
+    private final String commentMsg;
+    private final String userEmail;
+    private final long timestamp;
+    private final String imageURL;
```

Comment:
this looks like a PoJo (simple data holding class with no other methods). You might want to explore lombok (or google protobuf) and builders that they provide.
Much easier to work with and will save you a lot of boilerplate code

Reply:
Updated code with lombok

Reply:
use a @NoArgsConstructor with @Getter/@Setter as required and builder to initialise the object